.travis.yml

```diff
@@ -40,6 +40,8 @@ matrix:
       env: SYMFONY_REQUIRE=4.4.*
     - php: 7.4
       env: SYMFONY_REQUIRE=5.0.*
+    - php: nightly
+      env: COMPOSER_FLAGS="--no-interaction --ignore-platform-reqs" SYMFONY_PHPUNIT_VERSION=9 SYMFONY_REQUIRE=5.0.*
```

Comment:
is the `ignore-platform-reqs` because of using nightly instead of php 8? or is there a different reason why we need to ignore the platform?

Reply:
Because not all dependencies ready
```
/srv # php -v
PHP 8.0.0rc1 (cli) (built: Oct  1 2020 05:09:35) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.0-dev, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.0rc1, Copyright (c), by Zend Technologies

/srv # composer why-not php:8
symfony-cmf/routing   2.x-dev  requires  php (^7.2)  
doctrine/persistence  2.0.0    requires  php (^7.1)  
doctrine/reflection   1.2.1    requires  php (^7.1)  
symfony-cmf/testing   3.x-dev  requires  php (^7.1)  
```

Reply:
i started looking into allowing php 8 in the other systems. its a lot of repositories :scream: and aparently doctrine dbal is not ready for php 8, which will make things tricky as some of those things want to run tests with dbal...

i guess merging this PR when the dependencies are not php 8 ready does not achieve much? or do you still want to merge so that we can easier work on other php 8 readyness PRs?

Reply:
Personally I'd prefer to get this in to unblock further work on dependant packages

Reply:
Thank you a lot!